cpu/stm32/include/cpu_conf.h

```diff
@@ -116,6 +116,11 @@ extern "C" {
 #define FLASHPAGE_SIZE                  (128U)
 #endif
 
+#if defined(CPU_FAM_STM32L0) || defined(CPU_FAM_STM32L1) || \
```

Comment:
I think this PR still includes the other commits.

Reply:
It does. that is why its waiting for the other PR, since it will have a conflict anyway

Reply:
Ah I understood what you meant, that happened in the original PR, so they are removed now. Thanks for pointing this out.